src/simsopt/field/tracing.py

```diff
@@ -433,7 +433,7 @@ def plot_poincare_data(fieldlines_phi_hits, phis, filename, mark_lost=False, asp
             if data_this_phi.size == 0:
                 continue
             r = np.sqrt(data_this_phi[:, 2]**2+data_this_phi[:, 3]**2)
-            axs[row, col].scatter(r, data_this_phi[:, 4], marker='o', s=0.2, linewidths=0, c=color)
+            axs[row, col].scatter(r, data_this_phi[:, 4], marker='o', s=5, linewidths=0, c=color)
```

Comment:
If you disagree with the default I set here, please make it an optional argument to pass to the function

Reply:
I tried with the default option and I couldn't see any output. It might be the fact that I only had a few points. For these cases, it could actually be misleading to have a blank plot. As I'm a bit wary to change the function definition and its ramifications throughout, I'll revert it back to 0.2 and let you decide the best way to make it as an optional argument.